common/borders/borders-en.html

```diff
@@ -0,0 +1,125 @@
+---
+{
+    "title": "Borders",
+    "language": "en",
+    "altLangPage": "borders-fr.html",
+    "breadcrumbs":
+        [
+            {
+                "title": "GCWeb home",
+                "link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html",
+            },
+        ],
+    "secondlevel": false,
+    "dateModified": "2023-06-20",
+    "share": "true",
+}
+---
+
+<p>
+	Borders are commonly used to provide visual separation between elements,
+	highlight specific content, or create decorative effects. By adjusting the
+	border properties, you can customize the appearance of the borders to match
+	your desired design.&nbsp;
+</p>
+<div class="row">
+	<div class="col-md-6">
+		<h3 class="mrgn-tp-0 text-success">
+			<span class="glyphicon glyphicon-ok-circle"></span> Correct use
+		</h3>
+		<p>Compliance point(s):</p>
+		<ul>
+			<li>
+				Use to indicate focus on interactive elements, providing visual
+				feedback to users who navigate using the keyboard or assistive
+				technologies.
+			</li>
+			<li>
+				Ensure that the border color has sufficient contrast with the
+				background color to ensure visibility for users with visual
+				impairments or color vision deficiencies.
+			</li>
+			<li>
+				Use additional visual cues, such as color, icons, or text, to
+				supplement the meaning conveyed by borders.
+			</li>
+			<li>
+				Responsive design should be considered, ensuring that borders
+				adapt well to different screen sizes and zoom levels.
+			</li>
+		</ul>
+
+		<h3 class="mrgn-tp-0 text-danger">
+			<span class="glyphicon glyphicon-remove-circle"></span> Incorrect
+			use
+		</h3>
+		<p>Compliance point(s):</p>
+		<ul>
+			<li>
+				Responsive design should be considered, ensuring that borders
+				adapt well to different screen sizes and zoom levels.
+			</li>
+			<li>
+				Don't rely on color alone to distinguish borders. Color-blind
+				users may have difficulty perceiving color differences, so use
+				additional visual cues like texture, patterns, or labels.
+			</li>
+			<li>
+				Don't use excessive or distracting borders that may overwhelm or
+				confuse users, especially those with cognitive disabilities or
+				attention disorders.
+			</li>
+			<li>
+				Don't use borders as a substitute for proper semantic structure
+				and organization. Use appropriate HTML elements, headings, and
+				other structural elements to convey content hierarchy.
+			</li>
+		</ul>
+	</div>
```

Comment:
Please remove the Correct uses and the Compliance point, this is out-of-scope for GCWeb :)